internal/services/cognitive/cognitive_account_resource.go

```diff
@@ -591,6 +580,19 @@ func resourceCognitiveAccountRead(d *pluginsdk.ResourceData, meta interface{}) e
 			}
 			d.Set("local_auth_enabled", localAuthEnabled)
 
+			if localAuthEnabled {
+				keys, err := client.AccountsListKeys(ctx, *id)
+				if err != nil {
+					// note for the resource we shouldn't gracefully fail since we have permission to CRUD it
+					return fmt.Errorf("listing the Keys for %s: %+v", *id, err)
+				}
+
+				if model := keys.Model; model != nil {
+					d.Set("primary_access_key", model.Key1)
+					d.Set("secondary_access_key", model.Key2)
+				}
+			}
```

Comment:
Can you please update the documentation with a note below these properties with the explanation that these are only available if `location_auth_enabled` is `true`